### PR TITLE
Corrects cheesy typo

### LINKED ISSUE #27: Incorrect title on cheesy enchiladas' page
As a user, I should see the title of the cheesy enchiladas recipes as "cheesy enchiladas".

## Reproduction Steps
1. Go to the cheesy enchiladas recipe from the Mexican Section.
2. Observe that it says "Cheese Enchiladas" at top.

### PR DESCRIPTION
**Pull request by Amy-Hailu**
#fixes #27

### CODE REVIEW

File: src/routes/mexican/enchiladas/+page.svelte [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div>
-  <h1>Cheese Enchiladas</h1>
+  <h1>cheesy Enchiladas</h1>
```
Comment (bananabrann) [inline src/routes/mexican/enchiladas/+page.svelte:2]:
```suggestion
  <h1>Cheesy Enchiladas</h1>
```

File: src/routes/mexican/tamales/+page.svelte [hunk 2]
```diff
@@ -16,6 +16,8 @@
   <li>1 cup of chicken broth (may need more to get to the right consistency).</li>
   <li>1 tsp baking powder.</li>
   <li>1/2 cup of lard or shortening.</li>
+  <li>Salt to taste.</li>
+=======
```
Comment (bananabrann) [inline src/routes/mexican/tamales/+page.svelte:20]: Should not be included

**Review by bananabrann — CHANGES_REQUESTED** [review]

**Review by bananabrann — APPROVED** [review]

**Comment by bananabrann** [comment]
@Amy-Hailu **_squash_** merge

**Review by bananabrann — CHANGES_REQUESTED** [review]
PR is linked to the wrong issue. Please change it to #27

**Review by Amy-Hailu — COMMENTED** [review]
corrects enchiladas title typo

**Review by Chubbar5 — APPROVED** [review]

**Review by bananabrann — APPROVED** [review]

**Pull request merged**